Modules/Devices/test/Devices.Domain.Tests/Identities/StartDeletionProcessAsOwnerTests.cs

```diff
@@ -42,17 +43,35 @@ public void Start_deletion_process()
         auditLogEntry.NewStatus.Should().Be(DeletionProcessStatus.Approved);
     }
 
+    [Fact]
+    public void Start_deletion_process_as_owner_from_another_device()
+    {
+        // Arrange
+        var activeIdentity1 = CreateIdentity();
+        var activeIdentity2 = CreateIdentity();
+        var activeDevice1 = new Device(activeIdentity1);
+        var activeDevice2 = new Device(activeIdentity2);
```

Comment:
How can there be multiple active Identities and multiple active Devices?
=> You should really improve the naming